rs/src/parse/expressions/literals.rs

```diff
@@ -670,7 +670,7 @@ fn parse_fixed_char(
         data_type::Compound::FixedChar,
         nullable,
         variation,
-        vec![x.len() as u64],
+        vec![x.len() as i64],
```

Comment:
It's possible that `x.len().try_into().expect("can't cast or some better message")` may be useful here to prevent bizarre boundary cases where a `usize` value is larger then `i64::MAX` 🤷🏻

Reply:
Fair point... though dealing with `Vec::len()`s that imply that the vector is larger than 9 exabytes feels like sufficient cause for a bare `unwrap()` :sweat_smile: